source/frontend/src/utils/propertyUtils.ts

```diff
@@ -81,6 +81,20 @@ export const formatApiAddress = (address?: Api_Address) => {
   return values.join(' ') + (address?.postal ? ', ' + (address?.postal ?? '') : '');
 };
 
+/**
+ * Provides a formatted address as a string.
+ * @param address Address object from property.
+ * @returns Civic address string value.
+ */
+export const formatApiSummaryAddress = (address?: Api_Address) => {
+  const values = [
+    address?.streetAddress1 ?? '',
+    address?.streetAddress2 ?? '',
+    address?.streetAddress3 ?? '',
```

Comment:
This method seems to be replacing **formatApiAddress** - any reason to create a new method instead of updating the old one?

Reply:
Agree, the more formats we have the more likely we are to make a mistake